programmers2/위장.js

```diff
@@ -0,0 +1,15 @@
+function solution(clothes) {
+    let answer = 1;
+    const clothesObj={};
```

Comment:
let 보다 const 먼저 선언해 주시고 컨벤션 잊지말아요~!!
```suggestion
    const clothesObj = {};
    let answer = 1;
```